test/jest.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from '@jest/globals';
+export { describe, expect } from '@jest/globals';
+
+export type Test = () => void;
+
+export function wait(name: string, delay: number) {
+    test(
+        `wait ${name}`,
+        async () => {
+            console.info(`Waiting: ${delay} milliseconds ${name}`);
+            await new Promise((resolve) => {
+                setTimeout(resolve, delay);
+            });
+            expect(true);
+        },
+        delay > 5_000 ? delay * 2 : 5_000 // There is nothing here that will cause the delay to be extreme and twice the delay is an easy amount that is hopefully guaranteed to be greater than the wait time
```

Comment:
calling this function takes a few more milliseconds than what is specified in the delay. Since we never want the delay to time out we will add an ample amount of extra time to the timeout. Since doubling a short delay may not give us enough extra time we will have a reasonable floor.

Reply:
delay > 2_500